token/core/cmd/pp/gen.go

```diff
@@ -90,8 +115,8 @@ func Cmd() *cobra.Command {
 	flags.StringSliceVarP(&Auditors, "auditors", "a", nil, "list of auditor keys in the form of <MSP-Dir>:<MSP-ID>")
 	flags.StringSliceVarP(&Issuers, "issuers", "s", nil, "list of issuer keys in the form of <MSP-Dir>:<MSP-ID>")
 
-	flags.Int64VarP(&Base, "base", "b", 100, "base field used by the dlog driver")
-	flags.IntVarP(&Exponent, "exponent", "e", 2, "exponent field used by the dlog driver")
+	flags.Int64VarP(&Base, "base", "b", 100, "tt is used to define the maximum quantity a token can contain as Base^Exponent")
+	flags.IntVarP(&Exponent, "exponent", "e", 2, "tt is used to define the maximum quantity a token can contain as Base^Exponent")
```

Comment:
What is `tt`?

Reply:
Thanks @mbrandenburger , I have moved to subcommands :)